charts/et-cos/Chart.yaml

```diff
@@ -7,5 +7,5 @@ maintainers:
   - name: HMCTS Employment Tribunals Team
 dependencies:
   - name: java
-    version: 4.0.11
```

Comment:
I think we need to update the version 0.0.21 to 0.0.22 too when we update the dependencies.

Reply:
Thanks. I do that.